### PR TITLE
#135 - project name from xml

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #135 
Added slice for search and class to obtain project name from xml.

### CODE REVIEW

File: src/main/java/com/artipie/pypi/http/PySlice.java [hunk 1]
```diff
@@ -88,7 +88,7 @@ public PySlice(final Storage storage, final Permissions perms, final Authenticat
                 new RtRulePath(
                     new ByMethodsRule(RqMethod.POST),
                     new BasicAuthSlice(
-                        new WheelSlice(storage),
+                        new LoggingSlice(new WheelSlice(storage)),
```
Comment (olegmoz) [inline src/main/java/com/artipie/pypi/http/PySlice.java:91]: @olenagerasimova do not think we should add logging on this level, it is going to duplicate logging we got in Artipie. If you need logging in tests, then I'd suggest to wrap `PySlice`

**Review by olegmoz — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please see one comment

**Comment by olenagerasimova** [comment]
@olegmoz thanks, agreed, I've removed logging from all over `PySlice`

**Review by olegmoz — APPROVED** [review]
@olenagerasimova thanks, all looks good

**Comment by 0crat** [comment]
Job `gh:artipie/pypi-adapter#137` is not assigned, can't get performer

**Pull request merged**